swan-lake/integration/supported-data-formats.md

```diff
@@ -9,11 +9,11 @@ intro: Ballerina supports the data formats listed below.
+| JSON        | Convert between (to/from) Ballerina records and JSON                                                                                                                                                                                      | <a href="https://ballerina.io/learn/by-example/converting-from-user-defined-type-to-json/ ">Ballerina By Example<a/> |
 | XML         | Convert between (to/from) Ballerina records and XML                                                                                                                                                                                       | <a href="https://lib.ballerina.io/ballerina/xmldata/latest">API Docs<a/>                                                                                                                                                                                                                                                           |
 | YAML        | Convert between (to/from) Ballerina records and YAML                                                                                                                                                                                      | <a href="https://lib.ballerina.io/ballerina/yaml/latest">API Docs<a/>                                                                                                                                                                                                                                                              |
 | TOML        | Convert between (to/from) Ballerina records and TOML                                                                                                                                                                                      | <a href="https://lib.ballerina.io/ballerina/toml/latest">API Docs<a/>                                                                                                                                                                                                                                                              |
-| CSV         | Read/Write data from/to CSV file                                                                                                                                                                                                          | <a href="https://ballerina.io/learn/by-example/io-csv/">Read/write CSV<a/>                                                                                                                                                                                                                                                         |
```

Comment:
Why the text is changed to Ballerina by example?

Reply:
That is to be consistent with the text `API Documentation` on the other rows.

Reply:
ack